src/components/compound/MenuTab/types.ts

```diff
@@ -0,0 +1,9 @@
+import type { HTMLAttributes, ReactElement } from 'react';
+
+interface MenuTabProps extends HTMLAttributes<HTMLDivElement> {
+  children: [ReactElement, ReactElement];
+  initialOnChild: string;
+  tabText: [string, string];
```

Comment:
이젠 children과 tabText는 2개의 값만 받을수있지 않고 더 많이 받을수 있게 된것같습니다.
```suggestion
  children: ReactElement[];
  initialOnChild: string;
  tabText: string[];
```
로도 가능하지 않을까용

Reply:
@AlangGY 이 부분은 딱 2개만 받는 경우 그 개수를 제한하는 의미로 튜플을 써봤습니다.
그런데 만약 여러 개를 받을 수 있다고 하면 말씀해주신대로 바꿔야할듯요